config/default.yml

```diff
@@ -3150,6 +3150,10 @@ Style/CommentAnnotation:
     - HACK
     - REVIEW
     - NOTE
+  EnforcedStyle: colon
```

Comment:
I think we don't need an enforced style for this - we can just have an boolean options like `RequireColon: true`.

Reply:
OK, my initial thinking here was to leave it a bit more open with enforced style/supported styles in case somebody wanted to have a different separator, like a dash for example (i.e. `TODO- some todo comment here`). But I'm fine changing it to a simple boolean value indicating if a colon is required if you think that is better.

Reply:
Sometimes we should just keep things simple, so let's go with the boolean value.